config/public-testnet.yaml

```diff
@@ -57,7 +57,7 @@ checks:
   pt-retrieval:
     options:
       chunks-per-node: 3
-      postage-amount: 140000000
+      postage-ttl: 24h
```

Comment:
since these batches are shared across the test because they have the same label, it's better to create large batches in terms of size and TLL. 
TTL should be two weeks, and depth 21 🤔

Reply:
also i don't see the postage-labels here?

Reply:
> also i don't see the postage-labels here?

if not defined, default is `test-label`